### ISSUE TITLE
Improve NTFS support

### ISSUE THREAD

**Issue by joachimmetz** [log2timeline/plaso#316]
* [ ] [Add ADS stream support](https://codereview.appspot.com/258650043)
* [ ] [Handle certain files in special ways](https://github.com/log2timeline/plaso/issues/199)
* [ ] [Create a $MFT metadata file parser](https://github.com/log2timeline/plaso/issues/124)

**Comment by pettai** [comment]
I don't know if it's a separate bug/issue or if it relates to any of the improvements here, but I noticed that plaso doesn't seems to create an event for filenames with odd characters, like "svchost.exeÂ" or "svchost.exe?" (then extracted from a memory dump)

**Comment by joachimmetz** [comment]
> I don't know if it's a separate bug/issue

Yes that's a separate issue, I've created https://github.com/log2timeline/plaso/issues/323

**Comment by joachimmetz** [comment]
1.4 release items finished, moving to 1.5

**Comment by joachimmetz** [comment]
bumped milestone this will require more work

**Comment by Onager** [comment]
Not going to make release, removing milestone.

Please do not re-assign to a milestone until this issue has been cleaned up.

**Comment by MariasStory** [comment]
What about the "full path hints"?

**Comment by Onager** [comment]
Please do not re-assign to a milestone until this issue has been cleaned up.
